continuousflex/protocols/protocol_deep_hemnma_infer.py

```diff
@@ -114,6 +112,7 @@ def performDeepHEMNMAStep(self):
         params = " %s %s %s %d %d %d %d" % (self.imgsFn, weights, self._getExtraPath(), num_modes, batch_size, mode, device)
         script_path = continuousflex.__path__[0]+'/protocols/utilities/deep_hemnma_infer.py'
         command = "python " + script_path + params
+        command = Plugin.getContinuousFlexCmd(command)
```

Comment:
If I understand well, we have to activate the conda env each time we call an external program ?

Reply:
Hi Remi
I am glad everything worked for you!
If you want to use a package that is installed on the conda environment, then yes. But I think for Genesis, this is not necessary. It might be that Genesis (like the NMA binary) will benefit from lapack, arpack and blas libraries installed on the conda environment and lined to scipion. So I believe that you don't need to make any changes unless something doesn't work.

Reply:
Genesis is using lapack from continuousflex. I think is preferable to activate the environment  to avoid linking errors at runtime.